### DISCUSSION TITLE
Thank You!

### DISCUSSION THREAD

**Discussion by ghost** [nlohmann/json#2923] [General]
I recently implemented json.hpp single-file into my application and it was just an absolute dream! I replaced a custom trivial text format I'd been parsing by hand with a JSON Document Object Model reader class and it was simple, elegant, and effective with no difficulty implementing it based on a few simple samples that you provided here. Thanks for making the software world a better place NLOHMANN!

**Comment by nlohmann** [comment]
Thanks a lot! If you want to support me and my work on this project, you can become a sponsor on GitHub (https://github.com/sponsors/nlohmann), or use Paypal (http://paypal.me/nlohmann).
